src/description/eth_call.md

```diff
@@ -0,0 +1,15 @@
+* **MUST NOT** affect on-chain state 
+
+* **MUST** accept a `from` account that does not exist on-chain and if `from` is not defined it **MUST** be set to `0x0000000000000000000000000000000000000000`
+ 
+* If the `to` is null or not defined on-chain it **MUST** return an empty hex string
+
+* If `max_fee_per_gas` or `max_priority_fee_per_gas` is set the other must be set manually, otherwise they both **MUST** be set to the `gasPrice` or 0 when `gasPrice` is null
+
+* **MUST** accept `gas` that is greater than or equal to the minimum price for the current transaction call and less than `2^64 - 1` or `0xffffffffffffffff` 
```

Comment:
This can be captured by the specification, so shouldn't be included as an edge case.  Also, it is undecided whether this is capped at `2^64-1` or `2^53` (there are two open competing EIPs).  Also, "price" is not the right word to use here.  I think you mean "cost", but gas cost prior to execution is unknown so that cannot be validated by the client.